README.md

```diff
@@ -54,7 +54,9 @@ var styles = StyleSheet.create({
 
 ## Props
 
-- **`elevation`** _(Integer)_ - Must be between 0 and 5. The elevation you would like the elevated view to appear at.
+- **`elevation`** _(Integer)_ - Must be 0 or above, but anything higher than 24 is not officially supported.* The elevation you would like the elevated view to appear at.
```

Comment:
nit: lets just say 

Must be between 0 and 24. The elevation you would like the elevated view to appear at.